python/loops-lists/lists.py

```diff
@@ -0,0 +1,29 @@
+# Python Course
+# Lists
+
+# make a list of names:
+names = ["Joe", "Mary", "Eugene", "Katrina"]
+
+# print the whole list:
+print("here is a list of names", names)
+
+# print the length of the list:
+print("the length of the list is", len(names))
```

Comment:
similarly, explanation here about "why can we concatenate an int to a string with a comma but not with a plus sign"?